conceptnet5/vectors/cli.py

```diff
@@ -273,29 +270,19 @@ def run_miniaturize(input_filename, extra_vocab_filename, output_filename, k):
 @click.argument('output_dir', type=click.Path(writable=True, dir_okay=True))
 @click.argument('concepts_filename', type=click.Path(readable=True, dir_okay=False))
 @click.option('-l', '--language', default='en')
-@click.option('--tree-depth', default=1000)
-@click.option('-v', '--verbose', is_flag=True)
-def export_background(input_filename, output_dir, concepts_filename, language,
-                      tree_depth, verbose):
+def export_background(input_filename, output_dir, concepts_filename, language):
```

Comment:
Where does the `language` parameter come in?

Reply:
Oh I see, it's just passed on to `make_big_frame`, which is the only thing that needs it.